src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -2112,6 +2278,7 @@ private void recordPeopleMessage(JSONObject message) {
     }
 
     private void pushWaitingPeopleRecord() {
+        if (hasOptedOutTracking()) return;
```

Comment:
it's fine but do you need to do this?
1. `sendAllPeopleRecords` does the check already
2. it only gets called by `identify` which does the check too

Reply:
yup! again, in someone adds other functionality it's hard to keep track of things. defensive programming will assure in this case that nothing breaks in a different release